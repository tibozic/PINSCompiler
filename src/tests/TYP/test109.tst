fun f(x: integer, y: logical, z: logical, k: string): integer = { while x>10: k }
