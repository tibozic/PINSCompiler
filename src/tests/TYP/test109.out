[1:1-1:82]: ERROR: Function return type doesn't match function body return type: `int`!=`void`

